### PR TITLE
Add event / me / hasAccess

### PR DESCRIPTION
**Pull request by brettski**
Event access check for member based on event status, membership and tickets

### CODE REVIEW

File: src/lib/checkMemberEventAccess.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  let memberStore;
+  try {
+    orderStore = orderLib(firestore);
+    eventStore = eventLib(firestore);
+    memberStore = memberLib(firestore);
+  } catch (err) {
+    return Promise.reject(err);
+  }
+
+  const memberFunc = memberStore.get(memberId);
```
Comment (csell5) [inline src/lib/checkMemberEventAccess.js:25]: aren't these 3 promises? should we wrap them up in a promise.all

File: src/lib/checkMemberEventAccess.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  const eventFunc = eventStore.get(eventId);
+  const allocationFunc = orderStore.findMeOrderAllocationsForEvent({
+    memberId,
+    eventId,
+  });
+
+  return Promise.all([memberFunc, eventFunc, allocationFunc]).then(data => {
```
Comment (csell5) [inline src/lib/checkMemberEventAccess.js:32]: yea you already called it above.

**Review by csell5 — CHANGES_REQUESTED** [review]
I think you have a promise chain wrong.

File: src/lib/checkMemberEventAccess.js (hunk 1 above)
Comment (brettski) [reply src/lib/checkMemberEventAccess.js:25]: Yes they are look 7 lines down. 
I added the functions to variables first then added them to a Promise.all to be run and resolved.

File: src/lib/checkMemberEventAccess.js (hunk 2 above)
Comment (brettski) [reply src/lib/checkMemberEventAccess.js:32]: Not called above, Promises sent to Promise.all() to be resolved

**Review by csell5 — APPROVED** [review]

**Pull request merged**